build/Dockerfile

```diff
@@ -134,6 +134,23 @@ RUN --mount=type=cache,target=/home/${USER}/.cache/pip,uid=${USER_UID} \
     # Cleanup the bdist whl file
     rm $(head bdist_name) /tmp/bdist_name
 
+## Enable Aimstack if requested via ENABLE_AIM
+
+ARG ENABLE_AIM
+
+# Need a way to keep this aim version in sync with pyproject.toml
+RUN if [ "$ENABLE_AIM" ] ; then \
```

Comment:
Also this formatting is incorrect and thus you see the [image build failure](https://github.com/foundation-model-stack/fms-hf-tuning/actions/runs/10054558543/job/27789378962?pr=261). We could expect for ENABLE_AIM to be set to the string true by doing 
```sh
RUN if [[ "$ENABLE_AIM" == "true ]]; then \
```
Or we could expect that is ENABLE_AIM is set to any value that we set it
```sh
RUN if [[ -n "$ENABLE_AIM" == "true ]]; then \
```

Reply:
Hey @anhuong sure I noticed the bug I was busy last 2 days hence did not get to it but fixed now.